docs/_dg/TaggingFunctionality.md

```diff
@@ -0,0 +1,47 @@
+<!-- markdownlint-disable-file first-line-h1 -->
+### Tagging functionality
```

Comment:
Ditto, where is this being used?

Reply:
I added this under the Implementation.md